src/operations.js

```diff
@@ -12,16 +12,20 @@ import { reviewer } from './Reviewer'
 import { setter } from './Setter'
 import { traversal } from './Traversal'
 
-// combine two previews
+// different combinations of operations
+const combineGets = (g1, g2) => (x) => g2(g1(x))
+const combineSets = (o1, s2) => (v, x) => o1((inner) => s2(v, inner), x)
+const combineOvers = (o1, o2) => (f, x) => o1((inner) => o2(f, inner), x)
```

Comment:
Can you please provide tests for these simple combinators? Doing it should be straight forward and will coverage back up 😉

Reply:
Nope, these are just local functions which should not be exposed as part of the public API, something I would need to do to be able to test them. Furthermore, those lines are 100% covered by the tests, in all four categories.

Maybe we could make an exception this time, since we only cover 0.03% fewer lines, and #28 will include way more tests?